src/viam/app/data_client.py

```diff
@@ -733,6 +1064,24 @@ async def streaming_data_capture_upload(
     ) -> str:
         """Uploads the metadata and contents of streaming binary data.
 
+        ::
+
+            time_requested = datetime(2023, 6, 5, 11)
+            time_received = datetime(2023, 6, 5, 11, 0, 3)
+
+            file_id = await data_client.streaming_data_capture_upload(
+                data="byte-data-to-upload",
+                part_id="INSERT YOUR PART ID",
+                file_ext="png",
+                component_type='motor',
+                component_name='left_motor',
+                method_name='IsPowered',
+                data_request_times=[(time_requested, time_received)],
+                tags=["tag_1", "tag_2"]
+            )
+
+            print(file_id)
```

Comment:
This is the only place in the docs that I've seen a `print` statement. Maybe remove it for consistency?